.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Summary
+[Summarize your changes]
+
+## How would you rate the complexity of this PR? (Easy, Medium, Hard)
+[Review complexity]
+
+## Verification
+
+To verify this PR, you can:
+
+* Make sure that your PR follows the [contributing guidelines](CONTRIBUTING.md) and the [coding guidelines](https://github.com/ggerganov/llama.cpp/blob/master/README.md#coding-guidelines)
+* Test your changes using the commands in the [`tests`](tests) folder
+    * For instance, running the `./tests/test-backend-ops` command tests different backend implementations of the GGML library
+    * 
+
+* Execute [the full CI locally on your machine](ci/README.md) before publishing
```

Comment:
I would like the PR template to be very short, basically just this:

```
[x] I have read CONTRIBUTING.md
```

The reason is that seasoned contributors already know how to make PRs and what to include in the description, so no need for much text here.

We can then move the Contributing and Coding Guidelines from the README to the CONTRIBUTING file